tools/cli/main.go

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
copyright.

Reply:
Removed template copyright comment - already have LICENSE in the root of the repo